requirements.txt

```diff
@@ -2,6 +2,7 @@ lusid-drive-sdk-preview
 lusid-notifications-sdk-preview
 finbourne-sdk-utilities
 dve-lumipy-preview
+lusid-sdk-preview==0.11.4349

```

Comment:
What is this needed for? Do the queries need it to run? I thought they were just queried against CI?